tests/auth.setup.ts

```diff
@@ -0,0 +1,39 @@
+import path from 'path';
+
+import { test as setup, expect } from '@playwright/test';
+
+const authFile = path.join(__dirname, '../playwright/.auth/user.json');
+
+setup('authenticate', async ({ page }) => {
+  // Perform authentication using the app's email login flow (same as tests/integration/authentification/login.spec.ts)
+  // Uses environment variables TEST_USER_EMAIL and TEST_USER_PASSWORD. The test will be skipped
+  // if they are not provided by the CI/local environment.
+
+  // Open the login page (Playwright will resolve relative URLs if baseURL is configured)
+  await page.goto('/login');
+
+  // Click on the "Continue with Email" button
+  const authButtons = page.getByTestId('auth-buttons');
+  const continueWithEmailButton = authButtons.getByText('Email');
+  await continueWithEmailButton.click();
+
+  // Fill in credentials from environment variables
+  if (!process.env.TEST_USER_EMAIL || !process.env.TEST_USER_PASSWORD) {
```

Comment:
Please add these to `.env.example` and also in the body of the PR to be visible for the person responsible for deployments.